api/http/handler/stacks/stack_delete.go

```diff
@@ -169,5 +170,5 @@ func (handler *Handler) deleteStack(stack *portainer.Stack, endpoint *portainer.
 		return handler.SwarmStackManager.Remove(stack, endpoint)
 	}
 
-	return handler.ComposeStackManager.Down(stack, endpoint)
+	return handler.ComposeStackManager.Down(context.Background(), stack, endpoint)
```

Comment:
tip: `context.TODO()` will be a better fit here since at some point we will propagate the cancellable request context down.
This does not have an effect at the moment; however would make things easier in future - e.g. we could search by `context.TODO()` to identify all such cases which requires request context propagation.